context/app/static/js/components/Detail/ProvTabs/utils.js

```diff
@@ -0,0 +1,11 @@
+export function checkDataTypesForValues(dataTypes, valuesToCheck) {
+  return (
+    dataTypes.filter(
+      (type) => valuesToCheck.filter((value) => type.toLowerCase().includes(value.toLowerCase())).length > 0,
```

Comment:
```suggestion
      (type) => valuesToCheck.filter((value) => type === value).length > 0,
```
In some contexts, "be liberal in what you accept" is the right approach... but here I think we want to be conservative... otherwise we case-normalize in some places and not in others, and instead of cleaning up the data, people think something is wrong with the UI when a bug crops up.

Reply:
Also, use [some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) instead of checking list length.